### DISCUSSION TITLE
Bad rendering of TUI tools in terminal

### DISCUSSION THREAD

**Discussion by kfollesdal** [zed-industries/zed#15503] [Feature Requests]
I have understood that the terminal in Zed is using Alacritty.  So I have tried to run some TUI tools in the terminal, for example lazygit and btop. But the rendering is not as as expected and similarly to how it is in a normal Alacritty terminal.

Have anybody experience the same? Any settings that solve this?

Lazygit in Zed terminal:
<img width="1024" alt="Screenshot 2024-07-30 at 21 25 54" src="https://github.com/user-attachments/assets/fba04fd8-7f49-454c-b2aa-95d1bf888999">

Lazygit in Alacritty:
<img width="1000" alt="Screenshot 2024-07-30 at 21 25 38" src="https://github.com/user-attachments/assets/9b7dd59b-4dc0-4c4c-bd0f-327d7a8ef0a9">

Zed 0.145.1 tried on both Mac and Linux

**Comment by mosenturm** [comment]
For me TUI is ok.
Zed from Git, Ubuntu 22.04 Gnome Wayland
![image](https://github.com/user-attachments/assets/c6bee198-d803-435e-ae8c-a911e109530e)

**Comment by mikayla-maki** [comment]
Adding this to your settings will fix it :)

```json
{
  "terminal": {
    "line_height": "standard"
  }
}
```

**Reply by kfollesdal** [reply]
> Adding this to your settings will fix it :)
> 
> ```json
> {
>   "terminal": {
>     "line_height": "standard"
>   }
> }
> ```

Thanks, strangely did not work for me. So probably something with my setup. I will try to figure out what is going on. Let you know if I figure it out.

**Reply by dbedggood** [reply]
Did you figure it out?

**Reply by jesusvico** [reply]
I have the same problem. I am running Gemini CLI inside the Zed terminal and the TUI is not rendering correctly. This fix did not work for me.

**Reply by totoroyyb** [reply]
I don't think this is an actual fix, just making thing a bit less worse compared to the default "comfortable" line height. If increase the font size, there is still noticeable gaps between lines (say vertical bar). Tuning down line height indefinitely seems not to be a good solution either.
